settings.gradle

```diff
@@ -18,3 +18,19 @@ includeBuild('build-logic') {
     substitute module('app.cash.paparazzi:paparazzi-gradle-plugin') using project(':paparazzi-gradle-plugin')
   }
 }
+
+def brokenProjects = [
+  "appcompat-missing",
+  "invalid-application-plugin",
+  "missing-library-plugin",
+  "multiplatform-plugin-without-android",
+  "night-mode-xml", // TODO fix this one or remove it? Maybe left out of PaparazziPluginTest by accident?
+]
+
+new File(rootDir, "paparazzi-gradle-plugin/src/test/projects").eachDir {
```

Comment:
What an idea! Nice.

I think we would want to do this conditionally, because doing `gradlew build` or `gradlew test` locally to quickly check if Paparazzi is functioning would `build`/`test` these projects as well, which is double work since the `:p-g-p:test` will run them anyway.

If we wrap this whole added block in `val isIdea = (System.getProperty("idea.version") ?: "").isNotEmpty()` (or equivalent), they would only be included in AS/IDEA and not in command line/CI.

What do you think?

Reply:
love it.  @jrodbx wdyt?

Reply:
looking at the CI logs, it looks like now all these test projects would be subject to the check task running on them, which now does added work....

so, not sure if doubling down on this, if extra script hacking, to improve IDE syntax highlighting is the right tradeoff, here.

re: running tests in the IDE, def want this, but I suspect that can be accomplished a different way.

@JakeWharton thoughts?

Reply:
Presumably if we did @TWiStErRob's suggestion, those tests would no longer be run as part of ./gradlew check (since you're not including the projects unless you're in the IDE)

Reply:
I don't know anything about these nested builds for this project. In the projects I work on I rarely have to change anything in the nested builds. I would probably look into pursuing the ability to use `includeBuild` from the nested build so that you could open that project in the IDE and it would pull in the enclosing Paparazzi build with it. That way you could work within a single nested project at a time so that it models whatever scenario you want and then switch back to just the regular build when you're done.

Reply:
I guess each of them would need a settings.gradle to get that to work?  Would still be annoying to switch between projects, especially in the case where there's additional steps to the _actual_ test (e.g. `PaparazziPluginTest.rerunOnAssetChange`)

Is it really that big of a deal to add the IDE-only inclusion of these modules into the main one?

Reply:
¯\\\_(ツ)_/¯ Not my project so no idea. Was just asked for an opinion.

Reply:
yeah sorry, the second question was aimed at @jrodbx

Reply:
Ok, I like this much better.  Still a little annoying that the IDE adds the project name redundantly next to the module name, but I can live with it.

<img width="555" alt="Screen Shot 2023-07-18 at 5 15 21 PM" src="https://github.com/cashapp/paparazzi/assets/1868149/059856d3-f289-4e52-a700-5620221a96ab">